modules/cloudtrail/variables.tf

```diff
@@ -118,6 +118,27 @@ variable "cloudtrail_assume_role_policy_document" {
   EOF
 }
 
+
+variable "enable_cmdb_integration" {
+  type        = bool
+  default     = false
+  description = "Enable cmdb integration"
+}
+
+
+variable "cmdb_integration_config" {
+  type = object({
+    subscriptions = optional(list(object({
+      protocol               = optional(string, null)
+      endpoint               = optional(string, null)
+      endpoint_auto_confirms = optional(bool, false)
+    dead_letter_queue_arn = optional(string) })), [])
+    environment_variables = optional(map(any), {})
```

Comment:
in this case we know exact environment variables, so that please have them listed instead of wildcard map(any), it seem that you have no env variables so you can have it removed